cli/src/factory_ops/unboxing/network/management

```diff
@@ -0,0 +1,245 @@
+            _lerror "IP ${remote_mgmt_ip_srvnode_2} set on Server-B is not reachable over management network."
+            exit 21
+        )
+    )
+    
+    _linfo "Ping management IP ${remote_mgmt_ip_srvnode_1} of Server-A from Server-B to test sanity."
+    remote_execute ${pvt_ip_b} "ping -c1 ${remote_mgmt_ip_srvnode_1}" && ( 
+        _linfo "IP ${remote_mgmt_ip_srvnode_1} set on Server-A is reachable over management network." || (
+            _lerror "IP ${remote_mgmt_ip_srvnode_1} set on Server-A is not reachable over management network."
+            exit 21
+        )
+    )
+}
+
+
+function update_cluster_pillar_for_mgmt_ips {
+    ip_1=${1:-}
+    ip_2=${2:-}
+    gw_ip=${3:-}
+    netmask=${4:-}
+
+    # Configuration is static, update pillar with provided IPs
+    # If Configuration is DHCP. Set blank
+
+    if [[ -e "/srv/glusterfs/volume_prvsnr_data/srv/pillar/groups/all/uu_cluster.sls" ]]; then
+        # Get line number for management network section start for srvnode-1
+        occurance_1=$(grep -m2 -n mgmt_nw /srv/glusterfs/volume_prvsnr_data/srv/pillar/groups/all/uu_cluster.sls|head -1|cut -d: -f1)
+        _linfo "occurance_1: ${occurance_1}"
+        # Update pillar file
+        sed -in "${occurance_1},/public_ip_addr/ s/public_ip_addr:.*/public_ip_addr: \"${ip_1}\"/p" /srv/glusterfs/volume_prvsnr_data/srv/pillar/groups/all/uu_cluster.sls
+        sed -in "${occurance_1},/gateway/ s/gateway:.*/gateway: \"${gw_ip}\"/p" /srv/glusterfs/volume_prvsnr_data/srv/pillar/groups/all/uu_cluster.sls
+        sed -in "${occurance_1},/netmask/ s/netmask:.*/netmask: \"${netmask}\"/p" /srv/glusterfs/volume_prvsnr_data/srv/pillar/groups/all/uu_cluster.sls
+
+        # Get line number for management network section start for srvnode-2
+        occurance_2=$(grep -m2 -n mgmt_nw /srv/glusterfs/volume_prvsnr_data/srv/pillar/groups/all/uu_cluster.sls|tail -1|cut -d: -f1)
+        _linfo "occurance_2: ${occurance_2}"
+        # Update pillar file
+        sed -in "${occurance_2},/public_ip_addr/ s/public_ip_addr:.*/public_ip_addr: \"${ip_2}\"/p" /srv/glusterfs/volume_prvsnr_data/srv/pillar/groups/all/uu_cluster.sls
+        sed -in "${occurance_2},/gateway/ s/gateway:.*/gateway: \"${gw_ip}\"/p" /srv/glusterfs/volume_prvsnr_data/srv/pillar/groups/all/uu_cluster.sls
+        sed -in "${occurance_2},/netmask/ s/netmask:.*/netmask: \"${netmask}\"/p" /srv/glusterfs/volume_prvsnr_data/srv/pillar/groups/all/uu_cluster.sls
+
+        # Gluster would take care of this after unboxing
+        _linfo "pvt_ip_b: ${pvt_ip_b}"
+        scp -o "StrictHostKeyChecking=no" -i /root/.ssh/id_rsa_prvsnr /srv/glusterfs/volume_prvsnr_data/srv/pillar/groups/all/uu_cluster.sls "${pvt_ip_b}":/srv/glusterfs/volume_prvsnr_data/srv/pillar/groups/all/uu_cluster.sls
+    else
+        _lerror "Cluster pillar file for user data doesn't exist. Exiting..."
+        exit 41
```

Comment:
do we have some list of different codes and meanings?

Reply:
No meanings yet, I'm just adding them randomly for traceability sake.